### PR TITLE
refact: reduce blocking rendering

### PR DESCRIPTION
**Pull request by pftg**
Remove swipper.css from blocking other resources and added surge deploy scripts.

Now you can use `bin/setup` to setup macos dev env and `bin/surge/deploy` to deploy to surge some version.

## Summary by CodeRabbit

- **New Features**
	- Improved CSS loading strategy by separating critical and non-critical styles, enhancing page load performance.
	- Implemented non-blocking preload for non-critical CSS, optimizing initial rendering time.
	- Introduced automated setup and deployment scripts to streamline project configuration and deployment processes.
	- Added support for Bun in the GitHub Actions workflow to manage JavaScript dependencies.
	- Enhanced README documentation with clear instructions for local environment setup and staging deployment.

- **Bug Fixes**
	- Removed DNS prefetch directive to simplify resource loading and potentially improve performance.

- **Documentation**
	- Added fallback support for users with JavaScript disabled to ensure non-critical styles are applied.

### CODE REVIEW

File: themes/beaver/layouts/partials/page/testimonials.html
Comment (AucT) [inline themes/beaver/layouts/partials/page/testimonials.html:1]: Why not place swiper.min.css to non_critical_homepage.css?

Comment (pftg) [reply themes/beaver/layouts/partials/page/testimonials.html:1]: because it's not visible until document has been loaded

**Review by AucT — APPROVED** [review]

**Review by dgorodnichy — APPROVED** [review]
LGTM

**Comment by dgorodnichy** [comment]
@pftg , Conflicts.

**Review by AucT — COMMENTED** [review]
ok

**Pull request merged**